htdocs/societe/card.php

```diff
@@ -1656,7 +1674,7 @@ function init_check_no_email(input) {
 			print '<tr>';
 			print '<td>'.$form->editfieldkey('Currency', 'multicurrency_code', '', $object, 0).'</td>';
 			print '<td colspan="3" class="maxwidthonsmartphone">';
-			print $form->selectMultiCurrency(($object->multicurrency_code ? $object->multicurrency_code : $conf->currency), 'multicurrency_code', 1);
+			print $form->selectMultiCurrency((GETPOSTISSET('multicurrency_code') ? GETPOST('multicurrency_code') : $object->multicurrency_code), 'multicurrency_code', 1);
```

Comment:
Introducing the GETPOSTISSET seems good. But for the case test is false, we fallback on 
($object->multicurrency_code)
 where we should fallback on 
($object->multicurrency_code ? $object->multicurrency_code : $conf->currency)    -  like in old code
so if field was empty (never initialized in db and $object->multicurrency_code is null), we fallback on the global currency setup.